api/src/pcapi/core/educational/api/institution.py

```diff
@@ -182,18 +181,27 @@ def create_educational_institution_from_adage(institution: AdageEducationalInsti
     return educational_institution
 
 
-def get_institution_rural_level(
-    institution: EducationalInstitution | None,
-) -> str | None:
-    if not institution:
-        return None
-
-    try:
-        rows = list(InstitutionRuralLevelQuery().execute(institution_id=str(institution.id)))
-    except queries_base.MalformedRow:
-        return None
+def synchronise_rurality_level() -> None:
+    rows = list(InstitutionRuralLevelQuery().execute())
+    actual_rows = (
+        e._asdict()
+        for e in educational_models.EducationalInstitution.query.with_entities(
+            educational_models.EducationalInstitution.id, educational_models.EducationalInstitution.ruralLevel
+        )
+    )
```

Comment:
(détail) pour rendre la requête un peu plus lisible, peut-être :

```suggestion
    query = educational_models.EducationalInstitution.query.with_entities(
        educational_models.EducationalInstitution.id,    educational_models.EducationalInstitution.ruralLevel
    )
    actual_rows = (e._asdict() for e in query)
```